projects/1.16/assets/tinkers-construct/tconstruct/lang/zh_cn.json

```
@@ -1,2517 +1,2518 @@
+    "modifier.tconstruct.fiery": "怒火",
+    "modifier.tconstruct.fiery.flavor": "地狱烈火灌注其中！",
+    "modifier.tconstruct.fiery.description": "点燃攻击目标",
+    "modifier.tconstruct.necrotic": "噬生",
+    "modifier.tconstruct.necrotic.flavor": "汲取血液！",
+    "modifier.tconstruct.necrotic.description": "会榨取攻击目标的生命来滋养你！",
+
+    "_comment": "Damage boost",
+    "modifier.tconstruct.piercing": "穿刺",
+    "modifier.tconstruct.piercing.flavor": "变身仙人掌！",
+    "modifier.tconstruct.piercing.description": "工具会无视护甲造成额外伤害！",
+    "modifier.tconstruct.piercing.attack_damage": "穿刺伤害",
+    "modifier.tconstruct.smite": "亡灵杀手",
+    "modifier.tconstruct.smite.flavor": "死吧，贱种！",
+    "modifier.tconstruct.smite.description": "哪只亡灵不惧怕这金光闪闪的西瓜！",
+    "modifier.tconstruct.smite.attack_damage": "对亡灵生物伤害",
+    "modifier.tconstruct.bane_of_sssss": "嘶嘶杀手",
+    "modifier.tconstruct.bane_of_sssss.flavor": "你这吱吱蛛真不错啊！",
+    "modifier.tconstruct.bane_of_sssss.description": "工具会对蜘蛛、苦力怕和其他嘶嘶叫瘆死个人的虫子造成额外伤害！",
+    "modifier.tconstruct.bane_of_sssss.attack_damage": "对嘶嘶叫生物的伤害",
+    "modifier.tconstruct.antiaquatic": "水生杀手",
+    "modifier.tconstruct.antiaquatic.flavor": "基本上就是一根钓鱼竿了！",
+    "modifier.tconstruct.antiaquatic.description": "工具会对水栖生物造成额外伤害！",
+    "modifier.tconstruct.antiaquatic.attack_damage": "对水栖生物伤害",
+    "modifier.tconstruct.cooling": "火生杀手",
+    "modifier.tconstruct.cooling.flavor": "霜冻寒流！",
+    "modifier.tconstruct.cooling.description": "工具会对火基生物造成额外伤害！",
+    "modifier.tconstruct.cooling.attack_damage": "对火基生物伤害",
+    "modifier.tconstruct.sweeping_edge": "横扫之刃",
+    "modifier.tconstruct.sweeping_edge.flavor": "武士之道！",
+    "modifier.tconstruct.sweeping_edge.description": "横扫攻击所波及的目标将受到更高百分比的伤害",
+    "modifier.tconstruct.sweeping_edge.attack_damage": "横扫伤害",
+
+    "modifier.tconstruct.sharpness": "锋利",
+    "modifier.tconstruct.sharpness.1": "锋利",
+    "modifier.tconstruct.sharpness.2": "锋利II",
+    "modifier.tconstruct.sharpness.3": "锋利III",
+    "modifier.tconstruct.sharpness.4": "锋利IV",
+    "modifier.tconstruct.sharpness.5": "锋利V",
+    "modifier.tconstruct.sharpness.flavor": "多快的刀？",
+    "modifier.tconstruct.sharpness.description": "使工具锋芒逼人，摧枯拉朽！",
+
+    "_comment": "Armor",
+    "modifier.tconstruct.knockback_resistance": "击退抗性",
+    "modifier.tconstruct.knockback_resistance.flavor": "这可是个沉重的话题",
+    "modifier.tconstruct.knockback_resistance.description": "铁砧很重，所以它也能让你抵御击退，没问题吧？",
+    "modifier.tconstruct.golden": "金质",
+    "modifier.tconstruct.golden.flavor": "好似金子",
+    "modifier.tconstruct.golden.description": "让你的护甲金光灿灿，猪灵很喜欢它！",
+    "modifier.tconstruct.ricochet": "反弹",
+    "modifier.tconstruct.ricochet.flavor": "额外的弹力",
+    "modifier.tconstruct.ricochet.description": "受到攻击时反弹的更远，十分有利于逃跑",
+    "modifier.tconstruct.revitalizing": "振作",
+    "modifier.tconstruct.revitalizing.flavor": "能感受到，能量在流动",
+    "modifier.tconstruct.revitalizing.description": "能够给予更多血量",

```

```suggestion
    "modifier.tconstruct.revitalizing.description": "能够增加生命值上限",

```
此处已查证工匠大百科，是增加上限，也就是心数。